caddyhttp/httpserver/replacer.go

```diff
@@ -149,32 +157,59 @@ func (r *replacer) Replace(s string) string {
 	}
 
 	result := ""
+Placeholders: // process each placeholder in sequence
 	for {
-		idxStart := strings.Index(s, "{")
-		if idxStart == -1 {
-			// no placeholder anymore
-			break
+		var idxOffset, idxStart, idxEnd int
+
+		idxOffset = 0
```

Comment:
This variable always already equals 0 here because the line before declares it.

Reply:
I like the idea of having `idxOffset` obviously being set to zero just before the loop as a preference for code transparency, so I'd opt to remove it from the declaration instead, if that's suitable?